staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -37,6 +37,15 @@ const (
 	envBackoffDuration = "KUBE_CLIENT_BACKOFF_DURATION"
 )
 
+const (
+	// default configuration for exponentially backoff delay with jitter
+	// with an initial delay of 250ms, maximum delay of 1m and
+	// a maximum of 10% jitter.
+	defaultBackoffInitialDelay = 250 * time.Millisecond
+	defaultBackoffMaxDelay     = 1 * time.Minute
```

Comment:
1m sounds quite high to me - given we start from current 1s, I would try with something smaller first.

We generally set maxRetries to 10:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/rest/request.go#L153


Also @aojea comment:
https://github.com/kubernetes/kubernetes/pull/106272/#issuecomment-1120049668
is relevant here

I would probably decrease to sth like 10-15s for now.
[And as mentioned by @aojea - should be fixed in the release notes]

Reply:
okay, so i changed the initial delay to `1s` and max delay to `20s`. initial delay of `1s` to maintain current behavior - today by default we wait `1s` before retrying. 
https://github.com/kubernetes/kubernetes/blob/b53be1d66ef1c7f79410d619864d3788e084dd49/staging/src/k8s.io/client-go/rest/with_retry.go#L225-L229

I will remove the above with a follow up PR.